tests/flakyfunctional/database/00-simple.t

```diff
@@ -45,6 +45,7 @@ sed -i "s/$(cylc --version)/<SOME-VERSION>/g" "${NAME}"
 cmp_ok "${NAME}" << __EOF__
 UTC_mode|0
 cylc_version|<SOME-VERSION>
+icp|1
```

Comment:
Marking as draft, need to investigate this further

Reply:
The icp shouldn't be stored in the DB unless it was specified by the cli override, or if `now`/`next()` etc was used. Will fix this